diff --git a/dist/client.d.ts b/dist/client.d.ts
index 83927ca71c4684fc7e294d92b209322ea5784d09..1b52aa6759e38ec88cc0ed710e5801eb0aefee45 100644
--- a/dist/client.d.ts
+++ b/dist/client.d.ts
@@ -1,5 +1,3 @@
-import { Wallet } from "@ethersproject/wallet";
-import { JsonRpcSigner } from "@ethersproject/providers";
 import { type Chain, WalletClient } from "viem";
 import { IAbstractSigner } from "@polymarket/builder-abstract-signer";
 import { HttpClient } from "./http-helpers";
@@ -18,7 +16,7 @@ export declare class RelayClient {
     private readonly publicClient;
     readonly signer?: IAbstractSigner;
     readonly builderConfig?: BuilderConfig;
-    constructor(relayerUrl: string, chainId: number, signer?: Wallet | JsonRpcSigner | WalletClient, builderConfig?: BuilderConfig, relayTxType?: RelayerTxType, options?: RelayClientOptions);
+    constructor(relayerUrl: string, chainId: number, signer?: WalletClient, builderConfig?: BuilderConfig, relayTxType?: RelayerTxType, options?: RelayClientOptions);
     getNonce(signerAddress: string, signerType: string): Promise<NoncePayload>;
     getRelayPayload(signerAddress: string, signerType: string): Promise<RelayPayload>;
     getTransaction(transactionId: string): Promise<RelayerTransaction[]>;
